add-on/src/popup/browser-action/header.js

```diff
@@ -1,46 +1,53 @@
 'use strict'
 /* eslint-env browser, webextensions */
 
-const browser = require('webextension-polyfill')
 const html = require('choo/html')
 const logo = require('../logo')
 const powerIcon = require('./power-icon')
 const optionsIcon = require('./options-icon')
+const ipfsVersion = require('./ipfs-version')
 const gatewayStatus = require('./gateway-status')
 
 module.exports = function header (props) {
   const { ipfsNodeType, active, onToggleActive, onOpenPrefs, isIpfsOnline, onOpenWelcomePage } = props
   return html`
-    <div class="pt3 pb1 br2 br--top ba bw1 b--white" style="background-image: url('../../../images/stars.png'), linear-gradient(to bottom, #041727 0%,#043b55 100%); background-size: 100%; background-repeat: repeat;">
-      <div class="no-user-select">
+    <div class="br2 br--top ba bw1 b--white" style="background: linear-gradient(to bottom, #041727 0%,#043b55 100%);">
```

Comment:
I think ipfs-css has something like `ipfs-gradient-0` – could we reuse it here?

Reply:
Ah yes! Gradient direction is reversed but it's still a good luck. Thanks for remembering 😊